docs/src/api/class-browsercontext.md

```diff
@@ -49,6 +49,18 @@ page.goto("https://example.com")
 context.close()
 ```
 
+```csharp
+using var playwright = await Playwright.CreateAsync();
```

Comment:
shouldn't it be just `var playwright...` ?

Reply:
No, this is C#'s shorthand for doing a `using(var playwright...) { }`